### PR TITLE
fix shellcheck in several files

### PR DESCRIPTION
**Pull request by danielqsj**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix shellcheck failures in 
``` 
./pkg/kubectl/cmd/edit/testdata/record_testcase.sh
./pkg/util/verify-util-pkg.sh
./plugin/pkg/admission/imagepolicy/gencerts.sh
```

**Which issue(s) this PR fixes**:

work towards #72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
thanks. please squash the commits.
/priority backlog

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
@danielqsj PR needs rebase.

**Comment by danielqsj** [comment]
PR rebased.

@lavalamp would you mind to help review this ? thanks

File: plugin/pkg/admission/imagepolicy/gencerts.sh [hunk 1]
```diff
@@ -83,20 +83,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This file was generated using openssl by the gencerts.sh script
+// and holds raw certificates for the imagepolicy webhook tests.
+
+package imagepolic
```
Comment (liggitt) [inline plugin/pkg/admission/imagepolicy/gencerts.sh]: typo "imagepolic"

Comment (danielqsj) [reply plugin/pkg/admission/imagepolicy/gencerts.sh]: ohh, my fault. Thanks.
Fixed, PTAL

**Pull request merged**